integration-libs/opf/components/opf-checkout-billing-address-form/opf-checkout-billing-address-form.component.ts

```diff
@@ -11,28 +11,44 @@ import {
 } from '@spartacus/checkout/base/root';
 import { Address, Country, UserPaymentService } from '@spartacus/core';
 import { Card, ICON_TYPE } from '@spartacus/storefront';
-import { Observable, combineLatest } from 'rxjs';
+import { Observable, combineLatest, BehaviorSubject } from 'rxjs';
 import { tap, filter, map, shareReplay } from 'rxjs/operators';
 
 @Component({
   selector: 'cx-opf-checkout-billing-address-form',
   templateUrl: './opf-checkout-billing-address-form.component.html',
 })
 export class OpfCheckoutBillingAddressFormComponent implements OnInit {
+  iconTypes = ICON_TYPE;
+
   deliveryAddress$: Observable<Address | undefined>;
   countries$: Observable<Country[]>;
-  sameAsDeliveryAddress = true;
   showSameAsDeliveryAddressCheckbox$: Observable<boolean>;
-  userCustomBillingAddress: Address | undefined;
-  editBillingAddress = false;
-  iconTypes = ICON_TYPE;
+
+  sameAsDeliveryAddress$ = new BehaviorSubject(true);
+  userCustomBillingAddress$ = new BehaviorSubject<Address | undefined>(
+    undefined
+  );
+  editBillingAddress$ = new BehaviorSubject(false);
 
   constructor(
     protected checkoutDeliveryAddressFacade: CheckoutDeliveryAddressFacade,
     protected userPaymentService: UserPaymentService,
     protected checkoutPaymentService: CheckoutPaymentFacade
   ) {}
 
+  get sameAsDeliveryAddress(): boolean {
```

Comment:
`getValue()` does not subscribe to value changes of the observable which leaves us with a similar issue in detecting changes in the template. We should be using the async pipe with the observables in the template.